org-cyf-piscine/content/sprints/1/day-plan/index.md

```diff
@@ -0,0 +1,49 @@
++++
+title = 'day-plan'
+layout = 'day-plan'
+emoji= '🧑🏾‍🤝‍🧑🏾'
+menu_level = ['sprint']
+weight = 3
+[[blocks]]
+name="Induction"
+src="module/piscine/induction"
+time=15
+[[blocks]]
+name="Energiser"
+src="energisers/zip-zap-boing"
+[[blocks]]
+name="Morning break"
+src="blocks/morning-break"
+[[blocks]]
+name="Kickoff"
+src="module/piscine/kickoff"
+time=20
+[[blocks]]
+name="First group project"
+src="https://github.com/CodeYourFuture/The-Piscine/tree/main/Project-1"
+time=45
+[[blocks]]
+name="Development"
+src="module/piscine/development"
+time="60"
+[[blocks]]
+name="lunch"
+src="blocks/lunch"
+[[blocks]]
+name="Development"
+src="module/piscine/development"
```

Comment:
I'm finding it a little jarring to have the full block repeated so many times. I'd be tempted to split this into two blocks: A "first team development of the day" block as-is, and then a "... As above" block for the subsequent occurrences. WDYT?